### PR TITLE
next hackathon & training <3

### PR DESCRIPTION
**Pull request by FriederikeHanssen**

### CODE REVIEW

File: markdown/events/2023/hackathon-march-2023.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Hackathon - March 2023 (Online)
+subtitle:  A virtual online hackathon to develop nf-core together
+type: hackathon
+start_date: '2023-03-27'
+start_time: '10:00 CET'
```
Comment (mashehu) [inline markdown/events/2023/hackathon-march-2023.md:6]:
```suggestion
start_time: '10:00 CEST'
```

Comment (mashehu) [reply markdown/events/2023/hackathon-march-2023.md:6]: A classic 😆

Comment (FriederikeHanssen) [reply markdown/events/2023/hackathon-march-2023.md:6]: 🙈 it is already starting

File: markdown/events/2023/hackathon-march-2023.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: Hackathon - March 2023 (Online)
+subtitle:  A virtual online hackathon to develop nf-core together
+type: hackathon
+start_date: '2023-03-27'
+start_time: '10:00 CEST'
+end_date: '2023-03-29'
+end_time: '18:00 CET'
```
Comment (FriederikeHanssen) [inline markdown/events/2023/hackathon-march-2023.md:8]:
```suggestion
end_time: '18:00 CEST'
```

File: markdown/events/2023/training-march-2023.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: nf-core Training - March 2023
+subtitle: A set of global online Nextflow and nf-core training events
+type: workshop
+start_date: '2023-03-13'
+start_time: '05:00 CEST'
```
Comment (FriederikeHanssen) [inline markdown/events/2023/training-march-2023.md:6]:
```suggestion
start_time: '05:00 CET'
```

File: markdown/events/2023/training-march-2023.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+end_date: '2023-03-16'
+end_time: '21:30 CEST'
```
Comment (FriederikeHanssen) [inline markdown/events/2023/training-march-2023.md:8]:
```suggestion
end_time: '21:30 CET'
```

**Comment by FriederikeHanssen** [comment]
@nf-core-bot fix linting

File: markdown/events/2023/hackathon-march-2023.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+title: Hackathon - March 2023 (Online)
+subtitle:  A virtual online hackathon to develop nf-core together
```
Comment (FriederikeHanssen) [inline markdown/events/2023/hackathon-march-2023.md:3]:
```suggestion
subtitle:  A virtual hackathon with local gatherings to develop nf-core together
```

File: markdown/events/2023/hackathon-march-2023.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+end_time: '18:00 CEST'
+location_name: Gather town and Slack.
+import_typeform: true
+---
+
+# Welcome
+
+Save the date and join us for the March 2023 nf-core hackathon!
+
+This event is entirely virtual (we will be back in [gather.town](https://gather.town/)!) and we are attempting to support people in all timezones :earth_americas: :earth_asia: :earth_africa:
```
Comment (FriederikeHanssen) [inline markdown/events/2023/hackathon-march-2023.md:17]:
```suggestion
This event is virtual (we will be back in [gather.town](https://gather.town/)!) and as the last times we are attempting to support people in all timezones :earth_americas: :earth_asia: :earth_africa:. For this hackathon, we will try something new: hacking in-person is a lot of fun, so to get the best of both worlds, we are encouraging everyone to create small local groups, be it in your group, institute, or geographical region.
```

**Review by Emiller88 — APPROVED** [review]
LGTM!

**Review by ewels — APPROVED** [review]
Looks great, thanks!

File: markdown/events/2023/hackathon-march-2023.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
+---
+title: Hackathon - March 2023 (Online)
+subtitle: A virtual online hackathon to develop nf-core together
```
Comment (ewels) [inline markdown/events/2023/hackathon-march-2023.md:3]:
```suggestion
subtitle: A virtual hackathon with local gatherings to develop nf-core together
```

**Pull request merged**